src/mpid/ch4/src/ch4r_callbacks.c

```diff
@@ -257,6 +295,18 @@ static int recv_target_cmpl_cb(MPIR_Request * rreq)
         MPID_Request_complete(sigreq);
         /* Free the unexpected request on behalf of the user */
         MPIR_Request_free_unsafe(rreq);
+    } else if ((MPIDIG_REQUEST(rreq, req->status) & MPIDIG_REQ_RTS) &&
```

Comment:
Now we postponed the packing first segment into the RTS, this RTS is simply the old `MPIDIG_REQ_LONG_RTS`, right?

Reply:
see comments on `MPIDIG_SEND_LONG_REQ`